workstationsecuredrop-proxy-0.1.5-201912101723

```diff
@@ -0,0 +1,278 @@
+   dh_lintian -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_gconf -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_icons -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_perl -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_usrlocal -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_link -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_installwm -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_installxfonts -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   debian/rules override_dh_strip_nondeterminism
+make[2]: Entering directory '/home/user/debbuild/packaging/securedrop-proxy'
+find ./debian/ -type f -name '*.pyc' -delete
+find ./debian/ -type f -name 'pip-selfcheck.json' -delete
+find -type f -name RECORD -exec sed -i -e '/.*\.pyc.*/d' {} +
+dh_strip_nondeterminism override_dh_strip_nondeterminism
+make[2]: Leaving directory '/home/user/debbuild/packaging/securedrop-proxy'
+   dh_compress -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_fixperms -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_installdeb -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_gencontrol -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+dpkg-gencontrol: warning:     debian/changelog(l6): ignoring invalid week day 'Tues'
+LINE:  -- Allie Crevier <allie@freedom.press>  Tues, 18 Jun 2019 15:55:27 -0400
+dpkg-gencontrol: warning: Depends field of package securedrop-proxy: unknown substitution variable ${python3:Depends}
+   dh_md5sums -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+   dh_builddeb -O--python=/usr/bin/python3 -O--setuptools -O--index-url=https://dev-bin.ops.securedrop.org/simple -O--requirements=build-requirements.txt
+dpkg-deb: building package 'securedrop-proxy' in '../securedrop-proxy_0.1.4_all.deb'.
```

Comment:
two things: 
* securedrop-proxy_0.1.4_all.deb -> should be 0.1.5?
* my expectation is that the package should have the +buster or +stretch name in the version string (so there should be two build logs)

Reply:
should be fixed in the latest commit